digitalpalireader/content/js/dpr_pal.js

```diff
@@ -80,18 +80,7 @@ console.log('Loading DPR_PAL...');
   };
 
   DPR_PAL.chromeFileExists = fileLoc => {
-    var xmlhttp = new window.XMLHttpRequest();
-    try {
-      xmlhttp.open("GET", `${DPR_PAL.baseUrl}${fileLoc}`, false);
-      xmlhttp.onreadystatechange=function() {
-        xmlhttp.abort();
-      }
-      xmlhttp.send(null);
-    }
-    catch(ex) {
-      return false;
-    }
-    return true;
+    return $.ajax({type:"HEAD",url: `${DPR_PAL.baseUrl}${fileLoc}`,async: false}).status!=404;
```

Comment:
assuming the calls are equivalent, :heart: can you also please make the same change in xml.js?

Reply:
chromeFileExists() before this commit returned true for files that didn't exist too . 
Eg:

![bugwithchromefileexistsfirefox](https://user-images.githubusercontent.com/60284375/74107105-aabc5880-4b92-11ea-98a8-f1a0af700359.png)

> Environment : live-server hosting files , viewed on Firefox 72.0.2 (64-bit) on Ubuntu 18.04.3 LTS (Bionic Beaver)

this commit solves this while simplifying the definition with the help of jquery .

Reply:
chromeFileExists() before this commit returned true for files that didn't exist too . 
Eg:

![bugwithchromefileexistsfirefox](https://user-images.githubusercontent.com/60284375/74107105-aabc5880-4b92-11ea-98a8-f1a0af700359.png)

> Environment : live-server hosting files , viewed on Firefox 72.0.2 (64-bit) on Ubuntu 18.04.3 LTS (Bionic Beaver)

this commit solves this while simplifying the definition with the help of jquery .